javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+var fullName = 'Steve Behm';
+var isCool = true;
+var totalPets = 1;
+var unknown = null;
+var never;
+
+console.log('value of fullName: ', fullName);
+console.log('value of isCool: ', isCool);
+console.log('value of totalPets: ', totalPets);
+console.log('typeof fullName:', typeof 'Steve Behm');
+console.log('typeof isCool:', typeof true);
+console.log('typeof totalPets:', typeof 1);
```

Comment:
Instead of typing the value in your code directly, you should use the variable that holds that value.

Reply:
Okay sweet that makes sense